<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>950</width>
      <height>700</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="itinerary_tab">
     <attribute name="title">
      <string>Itinerary</string>
     </attribute>
     <widget class="QTableWidget" name="itinerary_table">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>621</width>
        <height>581</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>152</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name of Business</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type of Visit</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="officer_name_input">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Melody Henry</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Enter Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="officer_name_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name of Officer:</string>
      </property>
     </widget>
     <widget class="QLabel" name="grade">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Grade:</string>
      </property>
     </widget>
     <widget class="QLabel" name="period_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Period Start:</string>
      </property>
     </widget>
     <widget class="QLabel" name="area_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Area:</string>
      </property>
     </widget>
     <widget class="QLabel" name="parish_office_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parish Office:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="officer_grade_combo">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_entry">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>450</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>dd MMM yyyy</string>
      </property>
      <property name="date">
       <date>
        <year>2022</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
     <widget class="QComboBox" name="business_name_combo">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>490</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="visit_combo">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>530</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="date_entry_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
     <widget class="QLabel" name="business_name_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>490</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name of Business:</string>
      </property>
     </widget>
     <widget class="QLabel" name="visit_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>530</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type of Visit:</string>
      </property>
     </widget>
     <widget class="QLabel" name="itinerary_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>221</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>NIS INSPECTOR</string>
      </property>
     </widget>
     <widget class="QComboBox" name="parish_office_combo">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="area_combo">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="print_itinerary_btn">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>610</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Print Itinerary</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="period_start">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>290</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>dd MMM yyyy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submit_obj_btn">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>570</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="remove_row_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>570</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Row</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="load_itinerary_btn">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>610</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Itinerary</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>50</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/NIS_LOGO.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>400</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>INSERT TASK/OBJECTIVE</string>
      </property>
     </widget>
     <widget class="QLabel" name="period_label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Period End:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="period_end">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>330</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>dd MMM yyyy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reset_table_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>600</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset Table</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="report_tab">
     <attribute name="title">
      <string>Report</string>
     </attribute>
     <widget class="QTableWidget" name="report_table">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>249</y>
        <width>891</width>
        <height>371</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>123</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>123</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name of Business</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type of Visit</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ref. No.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Results</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>371</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Number of files carried forward from previous week:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>301</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Number of files received during the week:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Number of files cleared during the week:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>341</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Number of files on hand at the end of the week:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prev_work_entry">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="received_entry">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="total_entry">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="files_onhand_entry">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>140</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="files_cleared_entry">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>110</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="summary_table">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>501</width>
        <height>171</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>421</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>49</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>EFFECTTIVE</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>INEFFECTIVE</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>TOTAL</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>BEN.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>R.I.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>COMP.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>REG.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>EDUC.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>C/L</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>TOTAL</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SUMMARY OF WORK UNIT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="update_results_btn">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>630</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Results</string>
      </property>
     </widget>
     <widget class="QPushButton" name="print_report_btn">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>630</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Print Report</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>891</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QDateEdit" name="week_ended_date">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>dd MMM yyyy</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Week Ended:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
